### ISSUE TITLE
Percent missing data is off by a factor of 100 for "show" method of "genlight"

### ISSUE THREAD

**Issue by lvclark** [thibautjombart/adegenet#73]
``` 
/// GENLIGHT OBJECT /////////

 // 775 genotypes,  22,929 binary SNPs, size: 23.4 Mb
 4375463 (0.25 %) missing data
```
This is actually 25% missing data.
I've noticed this issue for a few years now, but thought I would finally report it since this weekend I'm looking at compatibility of adegenet 2.0 with polysat and various other R functions I've written.

**Comment by romunov** [comment]
It's probably missing a `* 100` [here](https://github.com/thibautjombart/adegenet/blob/52186788d3f75ff38caa0fb41ca6d7446269c80c/R/SNPbin.R#L521).

**Comment by thibautjombart** [comment]
Thanks. Fixed as of 0ee4c55360e9e35fbd28e87966db52937d11c64a
